rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc

```diff
@@ -1,6 +1,6 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="rosa-hcp-sts-creating-a-cluster-ext-auth"]
-= Creating ROSA with HCP clusters with external authentication
+= Creating a {hcp-title} cluster with direct authentication with an external OIDC identity provider
```

Comment:
I will say that there are 3 "with"'s in this heading. It's fine if that's what is needed, but I'd consider trying one of the other alternatives that doesn't have 3.

If you want to consider an alternative, perhaps even just putting "that uses" instead of the middle with would be better?

```suggestion
= Creating a {hcp-title} cluster that uses direct authentication with an external OIDC identity provider
```